astro-docs/src/content/docs/guides/Tips-n-Tricks/feature-based-testing.mdoc

```diff
@@ -0,0 +1,171 @@
+---
+title: Optimize Testing with Feature-Based Testing
+sidebar:
+  label: Feature-Based Testing
+description: Learn how to optimize your testing strategy by breaking down monolithic test projects into feature-scoped tests that run only when needed.
+filter: 'type:Guides'
```

Comment:
Honestly this is more of a concept than a guide, but it doesn't really fit into a 'core concept' cleanly, where we have kind of this 'dumping ground' of tips-n-tricks to place it in.